joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -981,6 +1074,15 @@ controller_interface::CallbackReturn JointTrajectoryController::on_activate(
     cmd_timeout_ = 0.0;
   }
 
+  {
+    if (get_update_rate() == 0)
+    {
+      throw std::runtime_error("Controller's update rate is set to 0. This should not happen!");
```

Comment:
Is this such a common error that we need to check it ? Wouldn't it make more sense to check this some where in ros2control when loading a controller + config?

Reply:
I always tend to add checks like that when I'm performing a division by a user-input variable. I don't expect this to ever happen, but it is out of my control at this point and if it ever happens, there will be a clear error.